samples/snippets/authenticate_service_account.py

```diff
@@ -30,6 +30,11 @@ def main():
         key_path, scopes=["https://www.googleapis.com/auth/cloud-platform"],
     )
 
+    # Alternatively, use service_account.Credentials.from_service_account_info() 
+    # to set credentials from a json string rather than read from a file
```

Comment:
It's not actually a string, though, right? I thought the "info" function takes a parsed value (dictionary).

Reply:
Right. I will correct that